docs/build/substrate-wasm.md

```diff
@@ -0,0 +1,215 @@
+# Substrate WASM Support
+
+We provide a custom data source processor for [Substrate WASM contract](https://ink.substrate.io/why-webassembly-for-smart-contracts). This offers a simple way to filter and index both WASM and Substrate activity on many Polkadot networks within a single SubQuery project.
+
+**Tested and Supported networks**
+
+| Network Name   | Websocket Endpoint                         | Dictionary Endpoint                                           |
+|----------------|--------------------------------------------|---------------------------------------------------------------|
+| Shiden         | `wss://shiden.api.onfinality.io/public-ws` | `https://api.subquery.network/sq/subquery/shiden-dictionary`  |
+| Shibuya        |                                            | `https://api.subquery.network/sq/subquery/shibuya-dictionary` |
+
```

Comment:
Any other network this could support in theory?

Reply:
Added one more network: Edgeware

Reply:
Thanks